lib/route.method.js

```diff
@@ -30,7 +31,10 @@ function createRouteInterceptor(routeMethod) {
         }
 
         function isAuthorizationRequired() {
-            return authorizationStrategy instanceof Function && routeMethod.authorizationRequired !== false;
+            var isMethodNotAllowed = methodNotAllowed === routeMethod.handlerFunc;
+            var authorizationEnabled = routeMethod.authorizationRequired !== false;
```

Comment:
Same as above -   
        
var authorizationEnabled = (routeMethod.authorizationRequired !== false); 
or 
var authorizationEnabled = !!routeMethod.authorizationRequired

is more readable.